src/main.cpp

```diff
@@ -0,0 +1,69 @@
+#include <stdio.h>
+#include <vector>
+#include <iostream>
+
+#define ADD 		'+'
+#define SUBSTRACT	'-'
+#define MULTIPLY	'*'
+#define DIVIDE		'/'
+
+//TODO: Dynamic typing, for now every value is int
+#define PseudoValue int // To enable dynamic typing we need to use one type for every value
+
+#define Program std::vector<ASTNode*>
+
+struct Runtime {
```

Comment:
Also could be good to create separate files for each struct here. What do you think?

Reply:
Sounds good :+1: